### PR TITLE
Identity Anchored Address Space

### PR DESCRIPTION
**Pull request by nugaon**
Describing Fair Data Society account and Identity Anchored Address Space (IAAS)

### CODE REVIEW

**Review by IgorShadurin — APPROVED** [review]
Terms definition looks good

File: text/0013-iaas.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+This approach employs widely-used concepts for wallet handling and assymetric keypair generation.
+
+The proposal aimed at setting the least amount of requirements for creating a Fair Data Society user for using FDS services.
+
+# Prior art
+- Book of Swarm v1.0 pre-release 7 - 2.2.3 Single-owner chunks 
+
+# Future possibilities
+The wallet handling is a crutial topic in this concern, but the login workflow will be elaborated in a different FIP.
+
```
Comment (tfius) [inline text/0013-iaas.md:55]: add option for subdomain registrations sub.username.fds ?

**Review by tfius — APPROVED** [review]

File: text/0013-iaas.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+The `BEE_API_URL` and `BEE_DEBUG_API_URL` are bee network configuration data for its APIs (e.g. 'http://localhost:1633', 'http://localhost:1635').
+Later it will be changed after Swarm introduces token based authentication on its endpoints.
+
+The wallet creation uses the [createRandom](https://docs.ethers.io/v5/api/signer/#Wallet-createRandom) function of the `ethers` library.
+
+This wallet can be used for Pods, folders creations and file uploads of the user.
+
+# Drawbacks
+TBD
```
[the remark this answers is not in the document]
Comment (nugaon) [reply text/0013-iaas.md:43]: what do you mean by that?

File: text/0013-iaas.md (hunk 1 above)
Comment (nugaon) [reply text/0013-iaas.md:55]: that would be more like the USNS (User Storage NameSpace). That can be elaborated in the mentioned FIP.

**Comment by tfius** [comment]
lets merge this then

**Pull request merged**